tt_metal/hw/inc/ethernet/dataflow_api.h

```diff
@@ -186,10 +351,75 @@ void eth_wait_for_bytes(uint32_t num_bytes) {
  */
 FORCE_INLINE
 void eth_receiver_done() {
-    erisc_info->user_buffer_bytes_sent = 0;
+    erisc_info->channels[0].bytes_sent = 0;
```

Comment:
we can make it a followup task to just remove these versions, before it gets used in too many other places.